obelisk_ws/src/cpp/obelisk_estimation_cpp/src/jointencoders_passthrough_estimator.cpp

```diff
@@ -0,0 +1,36 @@
+#include "rclcpp/rclcpp.hpp"
+
+#include "obelisk_estimator.h"
+#include "obelisk_ros_utls.h"
+
+class JointEncodersPassthrough : public obelisk::ObeliskEstimator<obelisk_estimator_msgs::msg::EstimatedState> {
```

Comment:
This class name doesn't match the python side. we need to pick and stick with a convention.